dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
 django-debug-toolbar==1.5     # https://django-debug-toolbar.readthedocs.io/en/stable/changes.html
+setuptools==33.1.1
```

Comment:
@jwalgran, @RobinIsTheBird, my app VM has `pip` 9.0.1, which appears to be the latest. So upgrading `pip` is not an option.

Did we try upgrading the django toolbar? We're on 1.5, but 1.7 is the latest.